### PR TITLE
6137 Payment Successful Name Request Modal UI

### PR DESCRIPTION
**Pull request by vysakh-menon-aot**
*Issue #:* /bcgov/entity#6137

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namerequest license (Apache 2.0).

### CODE REVIEW

File: client/src/components/payment/payment-complete-modal.vue [hunk 1]
```diff
@@ -150,3 +149,7 @@ export default class PaymentCompleteModal extends Mixins(NameRequestMixin, Payme
   }
 }
 </script>
+
+<style lang="scss" scoped>
+@import "@/assets/scss/base";
+</style>
```
Comment (severinbeauvais) [inline client/src/components/payment/payment-complete-modal.vue:155]: This style block is not used and could be deleted (optional).

File: client/src/components/payment/payment-confirm.vue [hunk 2]
```diff
@@ -26,25 +17,29 @@
...
+              <div class="fee-list__item-name">Payment Date</div>
+              <div class="fee-list__item-value">{{receipt.receiptDate}}</div>
+            </li>
+            <li class="container fee-list__item px-0" v-if="receipt">
+              <div class="fee-list__item-name">Amount</div>
+              <div class="fee-list__item-value">${{receipt.receiptAmount.toFixed(2)}} CAD</div>
+            </li>
+            <li class="container fee-list__item px-0" v-if="summary">
+              <div class='fee-list__item-name'>Status</div>
+              <div class='fee-list__item-value'>{{summary.statusCode}}</div>
```
Comment (severinbeauvais) [inline client/src/components/payment/payment-confirm.vue:36]: While you're here, change ' to " in the template strings (HTML).

**Review by severinbeauvais — APPROVED** [review]
Your code quality is excellent!

**Comment by severinbeauvais** [comment]
I ran this locally and took screenshots...

Confirm Name Request:

confirm name request

Payment Successful:

payment successful

**Comment by severinbeauvais** [comment]
The only thing I noticed is that the InVision shows "Your Name Request Number is XXX" in bold, but you only bolded the NR number.

Otherwise, looks bang on. Nice job!

**Comment by severinbeauvais** [comment]
I also noticed some comments on the InVision. Do they apply here?

https://projects.invisionapp.com/share/SZZVIQIB5RA#/screens/441895967/comments

**Comment by vysakh-menon-aot** [comment]
> I also noticed some comments on the InVision. Do they apply here?
> 
> https://projects.invisionapp.com/share/SZZVIQIB5RA#/screens/441895967/comments

Not sure about that. Will check with Tracey

**Comment by vysakh-menon-aot** [comment]
> The only thing I noticed is that the InVision shows "Your Name Request Number is XXX" in bold, but you only bolded the NR number.
> 
> Otherwise, looks bang on. Nice job!

Thats a miss from my side. Will fix it

File: client/src/components/payment/payment-complete-modal.vue (hunk 1 above)
Comment (vysakh-menon-aot) [reply client/src/components/payment/payment-complete-modal.vue:155]: I have used class `button-blue`, which is defined in base.scss thats why I imported it

Comment (severinbeauvais) [reply client/src/components/payment/payment-complete-modal.vue:155]: I still think it doen't need to be here -- global styles are automatically picked up as they are not scoped. But I could be wrong.

Comment (vysakh-menon-aot) [reply client/src/components/payment/payment-complete-modal.vue:155]: will test it

**Review by cameron-freshworks — APPROVED** [review]
Looks great! Merge when you are ready 👍

**Pull request merged**